schedule/schedule-appengine/src/test/java/com/jasify/schedule/appengine/util/FormatUtilTest.java

```diff
@@ -17,6 +19,20 @@
 
 public class FormatUtilTest {
 
+
+    @Before
+    public void before() {
+        // Your wondering why... well initializeDatastore sets the default timezone to UTC.
+        // FormatUtil.toString is called indirectly by other tests and the SimpleDateFormat is initialized with default timezone
+        // By the time we get here its UTC but your pc default timezone is probably not UTC so the tests will fail
+        TestHelper.initializeDatastore();
```

Comment:
The issue is that when TestHelper.initializeDatastore() is called the following code ends up being called
```
    public final LocalServiceTestHelper setUp() {
        this.originalDefaultTimeZone = TimeZone.getDefault();
        TimeZone.setDefault(this.timeZone);
```
When TestHelper.cleanupDatastore() is called the original timezone is restored.
START_FORMAT and FINISH_FORMAT are static and at initialisation they get a Calendar object which has the following initialisation
```
    private void initializeCalendar(Locale loc) {
        if (calendar == null) {
            assert loc != null;
            // The format object must be constructed using the symbols for this zone.
            // However, the calendar should use the current default TimeZone.
            // If this is not contained in the locale zone strings, then the zone
            // will be formatted using generic GMT+/-H:MM nomenclature.
            calendar = Calendar.getInstance(TimeZone.getDefault(), loc);
        }
    }
```
One of the FormatUtil tests exercises these static variables with a calendar object that has the default timezone.
```
    @Test
    public void testToStringActivity() throws Exception {
        Activity activity = new Activity();
        activity.setName("The Activity");
        GregorianCalendar calendar = new GregorianCalendar();
        calendar.setTimeZone(TimeZone.getDefault());
        calendar.set(1976, Calendar.JULY, 15, 22, 10);
        activity.setStart(calendar.getTime());
        calendar.add(Calendar.HOUR, 1);
        activity.setFinish(calendar.getTime());
        String formatted = FormatUtil.toString(activity);
        assertNotNull(formatted);
        assertEquals("The Activity [Thu, 15 Jul 22:10 - 23:10]", formatted);
    }
```
The issue is that if some other test that uses datastores runs earlier and triggers the initialisation of the START_FORMAT and FINISH_FORMAT variables they will be stuck at UTC but if FormatUtil does not change the default timezone to UTC its test will fail. I could change the test to also set the default timezone to UTC and then change back but this seemed simpler/easier/lasier